drivers/watchdog/Kconfig.nrf

```diff
@@ -0,0 +1,29 @@
+# Kconfig - nRF MDK WDT support
+#
+# Copyright (c) 2018 Nordic Semiconductor.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WDT_NRF
+	bool "nRF series Watchdog (WDT) Driver"
+	depends on SOC_FAMILY_NRF && CLOCK_CONTROL_NRF5
+	select HAS_DTS_WDT
+	select NRFX_WDT
+	default n
+	help
+	  Enable support for nRF MCU series WDT drivers
+
+config WDT_NRF_DEVICE_NAME
+	string "Device name for Watchdog (WDT)"
+	depends on WDT_NRF
+	default "WDT"
+	help
+	  Set the name used by WDT device during registration.
+
+config WDT_NRF_RELOAD_VALUE
+	int "Reload value in [ms]"
+	depends on WDT_NRF
+	default 2000
+	range 1 130150524
```

Comment:
This value is strange. I guess it was supposed to be: 0xFFFFFFFF (max value that can be written to `NRF_WDT->CRV`) * 30,51 us (1s / 32768) = 131039452 ms. But maybe we should limit it to some more reasonable value. 86400000 would be one day. Could there be a need to reload a watchdog less frequently?